Versions-mac.plist.in

```diff
@@ -33,6 +33,9 @@
 	<key>Features</key>
 	<array>
 		<string>sgen-concurrent-gc</string>
+		<string>activation</string>
+		<string>modern-http-client</string>
```

Comment:
^ that should be renamed since it's not the implementation we're using anymore (so it's a bit misleading even if it's based on modern. maybe `httpclient-nsurlsessionhandler` ?